src/aap_eda/settings/default.py

```diff
@@ -346,7 +346,7 @@ def _get_secret_key() -> str:
 # DEPLOYMENT SETTINGS
 # ---------------------------------------------------------
 
-DEPLOYMENT_TYPE = settings.get("DEPLOYMENT_TYPE", "local")
+DEPLOYMENT_TYPE = settings.get("DEPLOYMENT_TYPE", "podman")
```

Comment:
@hsong-rh Are we going to remove the local deployment with this PR or is that separate, since we are setting the default to be "podman" with this PR?

Reply:
@mkanoor The `local` deployment had been removed by https://github.com/ansible/eda-server/pull/370